lib/resources/styles.css

```diff
@@ -64,44 +71,44 @@ main {
   flex: 1;
   display: flex;
   flex-direction: row;
+  padding: 20px;
 }
 
 .sidebar-offcanvas-left {
-  flex: 0 1 195px;
+  flex: 0 1 230px;
   overflow-y: scroll;
   padding: 20px 0 15px 30px;
+  margin: 5px 20px 0 0;
 }
 
 ::-webkit-scrollbar-button{ display: none; height: 13px; border-radius: 0px; background-color: #AAA; }
 ::-webkit-scrollbar-button:hover{ background-color: #AAA; }
 ::-webkit-scrollbar-thumb{ background-color: #CCC; }
 ::-webkit-scrollbar-thumb:hover{ background-color: #CCC; }
 ::-webkit-scrollbar{ width: 4px; }
+::-webkit-overflow-scrolling: touch;
+
 .main-content::-webkit-scrollbar{ width: 8px; }
 
 .main-content {
   flex: 1;
-  overflow-y: auto;
-  padding: 20px 15px 0 30px;
-}
-
-.sidebar,
-.main-content {
-  margin: 20px 0;
+  overflow-y: scroll;
+  padding: 10px 20px 0 20px;
 }
 
 .sidebar-offcanvas-right {
   flex: 0 1 12em;
-  overflow-y: auto;
+  overflow-y: scroll;
   padding: 20px 15px 15px 15px;
+  margin-top: 5px;
   margin-right: 20px;
 }
 /* end for layout */
 
 body {
   -webkit-text-size-adjust: 100%;
   overflow-x: hidden;
-  font-family: "Source Sans Pro", sans-serif;
+  font-family: roboto, sans-serif;
```

Comment:
For similarity with below, `Roboto` (note the case)?

Reply:
I assume we're already importing Roboto from somewhere?

I see the import of Source Code Pro and the material icons font, but not of Roboto.

Reply:
Roboto is a builtin as discussed offline

Reply:
Fixed capitalization.